charts/opentelemetry-demo/values.yaml

```diff
@@ -9,10 +9,12 @@ global:
     kafka:
       enabled: true
 
-
 extensions:
   mysql:
     enabled: true
+    # Options to override the default image settings.
+    imageOverride:
```

Comment:
this config may not take effective, we need to refactor the logical in `mysql.yaml` template files. https://github.com/openinsight-proj/openinsight-helm-charts/blob/9753ec5bdc87637c924ab6daeb4e33fdb704e3d7/charts/opentelemetry-demo/templates/dataservice/mysql.yaml#L32